### PR TITLE
fix(react): adding babel jsx syntax for react lint

### PR DESCRIPTION
**Pull request by aqmnguyen**
Seems like JSX syntax isn't enabled, adding the babel JSX syntax looks to resolve this error. 

Error:
Screenshot 2021-05-13 at 14 56 12

### CODE REVIEW

File: eslint/react/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
   },
   extends: ['plugin:react/recommended'],
-  plugins: ['react'],
+  plugins: ['react', '@babel/plugin-syntax-jsx'],
```
Comment (alexlafroscia) [inline eslint/react/index.js:10]: This is interesting! I’m kind of surprised this works, passing a Babel plugin as an ESLint plugin.

Is there some documentation around on doing this?

Comment (aqmnguyen) [reply eslint/react/index.js:10]: Honestly, I thought it was a bit weird too but looking at the error message made me think it must 'work' (above screenshot kinda cut out the eslint msg), but below: 

Screenshot 2021-05-13 at 15 05 40

When I tested doing that in the local version it worked, I then tried it on the react ling config by doing a `yarn link` to ensure nothing else broke.

I went digging for docs around this but came up short, the link in the error msg is just a link to the plugin itself (https://git.io/vb4yA).

Comment (alexlafroscia) [reply eslint/react/index.js:10]: I think it's the plugins section of the _babel_ config that needs this added

**Comment by alexlafroscia** [comment]
`@movable/eslint-config-react@1.0.1` released with this change 👍

**Pull request merged**